### PR TITLE
Fix - Pdf order special characters

### PR DESCRIPTION
**Pull request by lguichard**
This PR fix special characters in the pdf order. Core fonts only support characters represented in Windows ANSI.

### CODE REVIEW

**Comment by alecritson** [comment]
@lguichard Not sure we should be specifying `DejaVu Sans` as it's not even a font I have installed so likely many others won't either.

**Comment by lguichard** [comment]
I fixed, use * {} directive fix the issue

**Comment by adevade** [comment]
@alecritson DejaVu Sans is [included with dompdf](https://github.com/dompdf/dompdf/tree/master/lib/fonts).

---

Also, @lguichard, the formatting seems wrong with double quotes wrapping most of the string: 

```css
font-family: "Helvetica Neue', 'Helvetica', Helvetica, Arial", sans-serif;
             ^                                              ^

/* SHOULD BE */
font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

(Not sure why Helvetica was included two times. And you only need quotes if font name includes space.)

## Suggestion

We could prepend the fallback list with DejaVu Sans, and it'll be used if found (which it always should be).

```css
font-family: 'DejaVu Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

**Comment by lguichard** [comment]
> @alecritson DejaVu Sans is [included with dompdf](https://github.com/dompdf/dompdf/tree/master/lib/fonts).
> 
> Also, @lguichard, the formatting seems wrong with double quotes wrapping most of the string:
> 
> ```css
> font-family: "Helvetica Neue', 'Helvetica', Helvetica, Arial", sans-serif;
>              ^                                              ^
> 
> /* SHOULD BE */
> font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
> ```
> 
> (Not sure why Helvetica was included two times. And you only need quotes if font name includes space.)
> 
> ## Suggestion
> We could prepend the fallback list with DejaVu Sans, and it'll be used if found (which it always should be).
> 
> ```css
> font-family: 'DejaVu Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
> ```

Ok, it seemed that the font type included in dompdf

**Comment by adevade** [comment]
Here's some more info about why DejaVu might be needed for international support:

> PDF documents internally support the following fonts: Helvetica, Times-Roman, Courier, Zapf-Dingbats, & Symbol. These fonts only support Windows ANSI encoding. In order for a PDF to display characters that are not available in Windows ANSI, you must supply an external font. Dompdf will embed any referenced font in the PDF so long as it has been pre-loaded or is accessible to dompdf and reference in CSS \@font-face rules. See the [font overview](https://github.com/dompdf/dompdf/wiki/About-Fonts-and-Character-Encoding) for more information on how to use fonts.
> 
> The [DejaVu TrueType fonts](https://dejavu-fonts.github.io/) have been pre-installed to give dompdf decent Unicode character coverage by default. To use the DejaVu fonts reference the font in your stylesheet, e.g. body { font-family: DejaVu Sans; } (for DejaVu Sans). The following DejaVu 2.34 fonts are available: DejaVu Sans, DejaVu Serif, and DejaVu Sans Mono.

https://github.com/dompdf/dompdf?tab=readme-ov-file#about-fonts--character-encoding

File: packages/admin/resources/views/pdf/order.blade.php [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <title>Order Invoice</title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
     <style>
-        body {
+        * {
             font-size:12px;
-            font-family:'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif;
+            font-family: "Helvetica Neue', 'Helvetica', Helvetica, Arial", sans-serif;
```
Comment (adevade) [inline packages/admin/resources/views/pdf/order.blade.php]:
```suggestion
            font-family: 'DejaVu Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

**Review by alecritson — APPROVED** [review]

**Review by alecritson — APPROVED** [review]

**Pull request merged**